moovingon_ai/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bb6dac8c-c4cd-414d-9b12-baef330a03a0",
+  "app_id": "moovingon-ai",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "moovingon.ai is a NOC orchestration and automation platform",
+    "title": "moovingon.ai",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "A moovingon.ai Dashboard",
+        "image_url": "images/moovingon.ai-dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
```

Comment:
Please include the following. Add relevant tags from this [list](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags): 
      "Category::<FILL IN>",
      "Offering::<FILL IN>",
      "Queried Data Type::<FILL IN if you are pulling data from datadog. otherwise, remove>",
      "Submitted Data Type::<FILL IN>"